lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -22,7 +22,7 @@
 /** @typedef {import('./report-renderer.js')} ReportRenderer */
 /** @typedef {import('./details-renderer.js')} DetailsRenderer */
 /** @typedef {import('./util.js')} Util */
-/** @typedef {'failed'|'manual'|'passed'|'not-applicable'} TopLevelClumpId */
+/** @typedef {'failed'|'manual'|'passed'|'notApplicable'} TopLevelClumpId */
```

Comment:
This doesn't have to change, but since it was a pass-through for the `scoreDisplayMode` values I thought it would make sense to update it.

Reply:
> This doesn't have to change

I think it does? Because the audits are clumped by their `scoreDisplayMode` (in `_getClumpIdForAuditRef`), so there wouldn't be a matching slot for the `notApplicable` ones if this wasn't updated

Reply:
I guess I knew that subconsciously and left it :wink: